bamboo-specs/bamboo.yml

```diff
@@ -0,0 +1,267 @@
+---
+version: 2
+plan:
+  project-key: CALC
+  key: CML
+  name: Calculation Module Latest
+stages:
+- Default Stage:
+    manual: false
+    final: false
+    jobs:
+    - Build and Test JDK 8
+    - Build and Test JDK 11
+    - Build and Test JDK 21
+    - Build and Test JDK 24
+    - Build and Test JDK 17
+- Deploy:
+    manual: false
+    final: false
+    jobs:
+    - Deploy to Maven
+- Release:
+    manual: true
+    final: false
+    jobs:
+    - Release to Maven
+Build and Test JDK 8:
+  key: BTJ8
+  tasks:
+  - checkout:
+      force-clean-build: 'false'
+      description: Checkout default repository
+  - script:
+      interpreter: SHELL
+      scripts:
+      - |-
+        #!/bin/bash -eu
+
+        set -x
+
+        export IMAGE="maven:3.9.9-amazoncorretto-8"
+
+        docker pull ${IMAGE}
```

Comment:
By the way, do we need to explicitly pull? Shouldn't run automatically do that?

Reply:
I like reviewing a smaller number of lines :)

Reply:
Should be a lighter review now with the latest commit... :-)